### ISSUE TITLE
Translation button

### ISSUE THREAD

**Issue by SotirisFtiakas** [AndreAle94/moneywallet#102]
I am currently finishing the translation of the app strings in Greek and was wondering if there is a translation button (can't find one) or it hasn't been implemented yet.

**Comment by AndreAle94** [comment]
Hi, thank you! What do you mean for translation button? The list of contributors is listed inside the 'about' activity. If you want to be credited, just add your name inside [this](https://github.com/AndreAle94/moneywallet/blob/master/app/src/main/assets/resources/about.json) file.

**Comment by SotirisFtiakas** [comment]
Hello, what I meant was if there is a button in the app that changes the language e.g. from English to Greek.

**Comment by AndreAle94** [comment]
No, there is no way to change the language inside the app. If you want to test another language you should change it in the android settings. I will consider this as a feature request.
